src/commands/src/shared/next-gen-stage.ts

```diff
@@ -0,0 +1,46 @@
+import { executeCliCommand, getCommitHash, getPatchId } from '../helpers';
```

Comment:
@branberry do we want to remove `getCommitHash` and `getPatchId` as they seem to be not be used in this file anymore.